std_reference_basic/src/contract.rs

```diff
@@ -58,207 +28,189 @@ pub fn execute(
     msg: ExecuteMsg,
 ) -> StdResult<Response> {
     match msg {
-        ExecuteMsg::TransferOwnership { new_owner } => {
-            try_transfer_ownership(deps, info, new_owner)
-        }
-        ExecuteMsg::AddRelayers { relayers } => try_add_relayers(deps, info, relayers),
-        ExecuteMsg::RemoveRelayers { relayers } => try_remove_relayers(deps, info, relayers),
+        ExecuteMsg::UpdateConfig { new_owner } => execute_update_config(deps, info, new_owner),
+        ExecuteMsg::AddRelayers { relayers } => execute_add_relayers(deps, info, relayers),
+        ExecuteMsg::RemoveRelayers { relayers } => execute_remove_relayers(deps, info, relayers),
         ExecuteMsg::Relay {
             symbols,
             rates,
             resolve_time,
             request_id,
-        } => try_relay(deps, info, symbols, rates, resolve_time, request_id),
+        } => execute_relay(deps, info, symbols, rates, resolve_time, request_id),
         ExecuteMsg::ForceRelay {
             symbols,
             rates,
             resolve_time,
             request_id,
-        } => try_force_relay(deps, info, symbols, rates, resolve_time, request_id),
+        } => execute_force_relay(deps, info, symbols, rates, resolve_time, request_id),
     }
 }
 
-pub fn try_transfer_ownership(
+pub fn execute_update_config(
     deps: DepsMut,
     info: MessageInfo,
     new_owner: Addr,
 ) -> StdResult<Response> {
-    let owner_addr = read_owner_store(deps.storage).load()?;
-    if deps.api.addr_canonicalize(&info.sender.as_str())? != owner_addr {
+    let mut config = CONFIG.load(deps.storage)?;
+    if info.sender != config.owner {
```

Comment:
Is it normal practice for contracts to add this for each function? 
It seems to duplicate for me. Not sure if it's good to create some function/macro. What do you think?